_posts/2019-10-30-upgrade-reminder-ru.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+заглавие: Убедитесь, что вы готовы к обновлению сети Bitcoin Cash
```

Comment:
I believe that the words "title" and "subtitle" should not actually be translated.

(It actually makes the Title render as "Upgrade Reminder Ru", rather than the Russian title and subtitle)